var/spack/repos/builtin/packages/libdap4/package.py

```diff
@@ -32,6 +32,12 @@ class Libdap4(AutotoolsPackage):
     depends_on("curl")
     depends_on("libxml2")
     depends_on("uuid")
+    depends_on("rpc")
+
+    def setup_build_environment(self, env):
+        # Configure script can insert library path here, resulting in a failure
+        if self.spec.satisfies("^libtirpc"):
+            env.set("TIRPC_LIBS", "-ltirpc")
```

Comment:
Shouldn't we add something based on `self.spec["rpc"]`? Here the `libdap4` package has knowledge about a specific provider or `rpc`, so it doesn't seem the right place to add that information.

Reply:
@alalazo - I'm happy to make changes, but I'm not sure what you are suggesting. Are you preferring something like this (or similar with a flag handler)?

```
env.set("TIRPC_LIBS", self.spec["rpc"].libs)
```

Reply:
Thanks for clarifying! I've made the change.